index.html

```diff
@@ -3,15 +3,37 @@
   <head>
     <meta charset="utf-8">
     <title>Weather API APP</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link href="css/styles.css" rel="stylesheet">
   </head>
 
   <body>
+    <div class="container-fluid">
+          <form id="search">
```

Comment:
@cbrooks2019 be careful re: indentation - it's off here. Incorrect indentation makes code harder to troubleshoot.